container/Dockerfile

```diff
@@ -0,0 +1,113 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
After rebase you'd see Dockerfile.centos and Dockerfile.rhel . 
So, probably for rhel8 we'd create Dockerfile.ubi8 ?